libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartSyncPlugin/SFSmartSyncPlugin.m

```diff
@@ -52,26 +51,25 @@ @interface SFSmartSyncPlugin ()
 
 @implementation SFSmartSyncPlugin
 
-- (SFSmartSyncSyncManager *)syncManager
+- (SFSmartSyncSyncManager *) syncManager
 {
     return [SFSmartSyncSyncManager sharedInstanceForStore:[SFSmartStore sharedStoreWithName:kDefaultSmartStoreName]];
 }
 
-- (SFSmartSyncSyncManager *)globalSyncManager
+- (SFSmartSyncSyncManager *) globalSyncManager
 {
     return [SFSmartSyncSyncManager sharedInstanceForStore:[SFSmartStore sharedGlobalStoreWithName:kDefaultSmartStoreName]];
 }
 
-- (void)resetSyncManager
+- (void) resetSyncManager
```

Comment:
Since this method is used for unit testing purposes only, should we move this to the test classes?